scripts/stanford-filter.script

```diff
@@ -94,6 +94,29 @@
    *[0018,1020].equalsIgnoreCase("")
   )
 
+  // Toshiba CT+PT - V6.06ER011
+  +( Manufacturer.equalsIgnoreCase("TOSHIBA")
+    * (Modality.equalsIgnoreCase("PT") + Modality.equals("CT"))
+    * (ManufacturerModelName.equalsIgnoreCase("AQUILION"))
+    * [0018,1020].equalsIgnoreCase("V6.06ER011")
+    * ImageType.contains("DERIVED")
```

Comment:
For regression testing. Testing files will likely be filtered out by line 1084, happened to me when I was testing SIEMENS. Need to twist the following lines.

+(!Modality.equals("MR") * ImageType.containsIgnoreCase("DERIVED"))